src/views/modals/AnnotationsModal/AnnotationsModal.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
```

Comment:
Was this copied over from console? We should avoid reimplementing console functionality and instead open an issue to expose the annotations modal.